src/Authenticator/AuthenticatorCollection.php

```diff
@@ -65,12 +65,12 @@ protected function _create($className, $alias, $config)
     /**
      * Resolves authenticator class name.
      *
-     * @param string $class Class name to be resolbed.
-     * @return string
+     * @param string $class Class name to be resolved.
+     * @return string|null
      */
     protected function _resolveClassName($class)
     {
-        return App::className($class, 'Authenticator', 'Authenticator');
+        return (string)App::className($class, 'Authenticator', 'Authenticator') ?: null;
```

Comment:
phpstan says:
> Method Authentication\Authenticator\AuthenticatorCollection::_resolveClassName() should return string|null but returns bool|string|null.

So I've added the type hint.

It's already becoming a PITA :(